README.md

```diff
@@ -1006,6 +1006,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Jasmin Carter](https://github.com/sleepykanguru22)
   - [Jason Dsouza](https://github.com/Jsn99)
   - [Jaswinder Singh](https://github.com/jaswindersingh2601)
+  - [Jaswanth Ch](https://github.com/jaswanth187)
```

Comment:
AFAIK, your name should be in line 1008 after Jason Dsouza (correct me if I'm wrong).
Let us know if you need any help with that. 🙂

Reply:
i also have a doubt, now my forked repo is behind few commits. now, what should be my approach?

Reply:
You can update your fork by clicking on a button similar to the pull request icon. The `main` branch should be updated, not the current one, because this branch doesn't need to be updated. Once you create another branch, it should be all updated.

Hope it's understandable. Let me know if you have any doubts. 🙂